packages/washboard/src/lattice/lattice-service.ts

```diff
@@ -67,7 +67,7 @@ class LatticeService {
   private connection?: NatsConnection;
 
   private config = {
-    latticeUrl: 'ws://localhost:4001',
+    latticeUrl: import.meta.env.VITE_NATS_WEBSOCKET_URL,
```

Comment:
Let's include a default if the env is not set
```suggestion
    latticeUrl: import.meta.env.VITE_NATS_WEBSOCKET_URL ?? 'ws://localhost:4001',
```

Reply:
You TypeScript people, always bragging about your `??` like you're so cool 😁

Reply:
I always love finding places to use the elvis operator `?:`

Reply:
Almost as cool as the Rust toilet bowl : `|_|()`